controller/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # build stage
-FROM golang:alpine AS build-env
+FROM arm64v8/golang:alpine AS build-env
```

Comment:
Hi @tiagoposse The build is failing because the architecture is not the expected one. Can you rollback this.

Reply:
Apologies for that, was forcing arm for my use case. This has been fixed and the PR rebased, hope this fixes it :)